src/ophyd_async/core/detector.py

```diff
@@ -59,11 +59,16 @@ class TriggerInfo(BaseModel):
     #: Sort of triggers that will be sent
     trigger: DetectorTrigger = Field()
     #: What is the minimum deadtime between triggers
-    deadtime: float = Field(ge=0)
+    deadtime: float | None = Field(ge=0)
```

Comment:
Would be good to mention that the behaviour when deadtime/livetime is None is that they are unchanged from the previous prepare that set those values.

Reply:
Could also make the Field(default=None) so that preparing for a step scan is less verbose